src/Compilers/CSharp/Portable/Syntax/RecordDeclarationSyntax.cs

```diff
@@ -8,16 +8,11 @@ namespace Microsoft.CodeAnalysis.CSharp.Syntax
 {
     public partial class RecordDeclarationSyntax
     {
-        internal SimpleBaseTypeSyntax? BaseWithArguments
+        internal PrimaryConstructorBaseTypeSyntax? PrimaryConstructorBaseTypeSyntax
```

Comment:
>PrimaryConstructorBaseTypeSyntax [](start = 51, length = 32)

To be consistent with the way properties on syntax nodes are named, you should drop the `Syntax` suffix.

Reply:
> To be consistent with the way properties on syntax nodes are named, you should drop the Syntax suffix.

Renamed

---
In reply to: [443925572](https://github.com/dotnet/roslyn/pull/45351#discussion_r443925572) [](ancestors = 443925572)